.storybook/preview-head.html

```diff
@@ -0,0 +1,3 @@
+<script>
+    window.global = window;
```

Comment:
What does this do?

Reply:
This is needed for some of the Storybook addons and the fix is suggested in https://github.com/storybookjs/storybook/issues/18399 and https://github.com/storybookjs/builder-vite/blob/main/packages/builder-vite/README.md#migration-from-webpack--cra